mne_qt_browser/_pg_figure.py

```diff
@@ -21,18 +21,26 @@
 from os.path import getsize
 
 import numpy as np
-from qtpy.QtCore import (
-    QEvent,
-    QThread,
-    Qt,
-    Signal,
-    QRectF,
-    QLineF,
-    QPointF,
-    QPoint,
-    QSettings,
-    QSignalBlocker,
-)
+
+try:
+    from qtpy.QtCore import (
+        QEvent,
+        QThread,
+        Qt,
+        Signal,
+        QRectF,
+        QLineF,
+        QPointF,
+        QPoint,
+        QSettings,
+        QSignalBlocker,
+    )
+except Exception as exc:
+    if exc.__class__.__name__ == "QtBindingsNotFoundError":
+        raise ImportError("No Qt binding found, please install pyqt5") from None
+    else:
+        raise exc
```

Comment:
IIRC the traceback is better this way, no extra stuff from reraising
```suggestion
        raise
```